do

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+echo
+echo "cleanup"
+[ -f test ] && rm test
+[ -f vt ] && rm vt
+echo
+echo "making V"
+./v -d debug_bounds_checking -o vt cmd/v
+echo
+echo "compiling test"
+./vt test.v -printfn main__main
+#echo "testing"
+#[ -f ./test ] && ./test
```

Comment:
this should be removed from the PR imho

Reply:
I agree, this is in a separate commit as I did not know where to place it.

Reply:
I agree, I added all the code so one could see how I used the test code. It should be removed.